cluster/addons/fluentd-gcp/README.md

```diff
@@ -0,0 +1,11 @@
+# Stackdriver Logging Agent
+==============
+
+Stackdriver Logging Agent is a DaemonSet which spawns a pod on each node
+that reads logs, generated by kubelet, container engine and containers
```

Comment:
I meant container runtime, thanks for noticing

As for the specifying, what logs fluentd collects, on the one hand I want to give an idea about all sources, on the other hand, I don't want to go into details, explaining that it collects logs from docker daemon, vm creation scripts, salt and so on. It's not a right place for such details IMO.